tests/shared/src/test/scala/matryoshka/data/fix.scala

```diff
@@ -20,20 +20,18 @@ import matryoshka.exp.Exp
 import matryoshka.helpers._
 import matryoshka.patterns.CoEnv
 import matryoshka.scalacheck.arbitrary._
-import matryoshka.specs2.scalacheck._
+import matryoshka.scalacheck.cogen._
 
 import scala.Int
 
+import org.scalacheck.Test
 import org.specs2.mutable._
 import scalaz._, Scalaz._
 import scalaz.scalacheck.ScalazProperties._
 
-class FixSpec extends Specification with CheckAll with AlgebraChecks {
-  "Fix" should {
-    "satisfy relevant laws" in {
-      checkAll(equal.laws[Fix[Exp]])
-    }
-  }
+class FixSpec extends Specification with AlgebraChecks {
+
+  equal.laws[Fix[Exp]].check(Test.Parameters.default)
```

Comment:
Ah, I thought I got rid of all the `addFragments` ones, but looks like I missed a few. The `.check` cases succeed when the `addFragments` ones sometimes give up on too many discarded cases. Also, the `.check` ones give better messages. `addFragments` loses the names of the `Prop`s.